### DISCUSSION TITLE
As I removed `alpha.nvim` for I never used it, how to bring the default startup screen texts back?

### DISCUSSION THREAD

**Discussion by vollowx** [ayamir/nvimdots#843] [Q&A]
I removed `alpha.nvim` because I have never used it (though maybe it's nice for new users), so I want to bring the default startup screen texts back in order to make it looks better. But how? I can't find the config.

* default startup screen texts means:

![image](https://github.com/ayamir/nvimdots/assets/73375859/3fc4c573-7c4b-4c21-8c06-38bb570b3887)

Also, when I press `<C-n>` to open `nvim-tree` straightly at the startup screen, it opens with the size of the all screen, but not just 30 chars as before.

**Comment by charliie-dev** [comment]
I think u might need to find events related to `BufEnter`, `WinEnter` and `BufWinEnter`. Not 100% sure, but it might be a direction.
